textdb/textdb-storage/src/main/java/edu/uci/ics/textdb/storage/TableMetadata.java

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.textdb.storage;
+
+import edu.uci.ics.textdb.api.schema.Schema;
+
+public class TableMetadata {
```

Comment:
Could you please add Jackson annotations to the `TableMetadata` class so that Jackson can automatically map the object to a JSON string?